src/scripts/components/audio/Audio.jsx

```diff
@@ -198,6 +200,30 @@ export default class Audio extends React.Component {
     });
   }
 
+  updateIframeSrc(audio) {
+    const { imageUrl } = this.state;
+    const { contributors, files, title } = audio;
+
+    const iframeSrcObj = {
+      audio: files["mp3_128"],
```

Comment:
Consider changing this to `url` or `src` so that it's more descriptive on the embed page's code

Reply:
Will do, thanks!